apps/prairielearn/src/pages/studentAssessments/studentAssessments.html.ts

```diff
@@ -35,7 +24,7 @@ export const StudentAssessmentsRowSchema = z.object({
   label: z.string(),
   credit_date_string: z.string(),
   active: AssessmentAccessRuleSchema.shape.active,
-  access_rules: AuthzAccessRuleSchema.array().nullable(),
```

Comment:
I believe this change is safe because of the coalesce with an empty array here:

https://github.com/PrairieLearn/PrairieLearn/blob/bf96affaace6514e83c62cd828ad2083749fea06/apps/prairielearn/src/sprocs/check_assessment_access.sql#L148-L155

This simplifies some downstream typing.